Dockerfile

```diff
@@ -3,26 +3,20 @@ MAINTAINER Tim Perry <pimterry@gmail.com>
 
 USER root
 
-RUN apt-get update && \
-    export DEBIAN_FRONTEND=noninteractive && \
-    # Set password to temp-password - reset to random password on startup
-    echo mysql-server mysql-server/root_password password temp-password | debconf-set-selections && \
-    echo mysql-server mysql-server/root_password_again password temp-password | debconf-set-selections && \   
-    # Instal MySQL for data, node as the JS engine for uglifier
-    apt-get install -y mysql-server nodejs
-    
-COPY . /opt/staytus
+ENV DEBIAN_FRONTEND="noninteractive"
 
-RUN cd /opt/staytus && \
-    bundle install --deployment --without development:test
-
-ENTRYPOINT /opt/staytus/docker-start.sh
+ADD docker-start.sh /docker-start.sh
+ADD . /opt/staytus/
```

Comment:
Minor point, but COPY is preferable to ADD: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy